queries.sql

```diff
@@ -6,4 +6,47 @@ SELECT name from animals WHERE (neutered = TRUE)AND(espace_attempts < 3);
 SELECT date_of_birth from animals WHERE (name = 'Agumon') OR (name = 'Pikachu');
 SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE NOT name = 'Gabumon';
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+  UPDATE animals SET species = 'unspecified';
+  select * from animals;
+ROLLBACK;
+select * from animals;
+
+BEGIN;
+  UPDATE animals SET species ='digimon' WHERE name LIKE '%mon';
+  select * from animals;
+  UPDATE animals SET species ='pokemon' WHERE species IS NULL;
+  select * from animals;
+COMMIT;
+
+BEGIN;
+  DELETE FROM animals;
+ROLLBACK;
+select * from animals;
+
+BEGIN;
+  DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+  select * from animals;
```

Comment:
- [ ] After deleting all animals born after Jan 1st, 2022, we have to create a savepoint for the transaction as seen in the requirements
![Screenshot from 2023-07-13 12-20-40](https://github.com/francksefu/vet_clinic/assets/46388113/503c95cd-c5ef-4aa7-b2f2-669e06a0837f)